src/oas2/transformers/__tests__/securities.test.ts

```diff
@@ -95,7 +95,7 @@ describe('securities', () => {
         name: 'a name',
       };
 
-      expect(translateToSecurities([basicSecurity, implicitSecurity, apiSecurity])).toMatchSnapshot();
+      expect(translateToSecurities([[basicSecurity, implicitSecurity, apiSecurity]])).toMatchSnapshot();
     });
   });
 });
```

Comment:
Please add some tests that will verify the `OR` case (return array of more than one arrays)